modules/network-observability-config-flows-at-ingestion.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-config-flows-at-ingestion_{context}"]
+
+= Filtering network flows at ingestion
+
+You can create filters to reduce the number of generated network flows. Filtering network flows can help reduce the resource usage of the Network Observability Operator.
```

Comment:
I wouldn't say "operator" here, it's more the other underlying components that are affected, not the operator itself.

```suggestion
You can create filters to reduce the number of generated network flows. Filtering network flows can help reduce the resource usage of the Network Observability components.
```